### ISSUE TITLE
Pico-Motor-Driver - need examples & documentation 

### ISSUE THREAD

**Issue by dsssssssss9** [waveshareteam/Pico_MircoPython_Examples#1]
Hi

Wiki for this product states that example code & full manual are available - they are not!!

Very confusing for beginners & students

Please can you provide this as a matter of haste

**Comment by EngineerWill** [comment]
hi,dsssssssss9 
Thank you for your feedback. What programming language do you usually use

**Comment by Bfreak** [comment]
@dsssssssss9 Yep, its a nightmare, waveshare and simcom both do not provide adequate data for their products.
